sys/include/architecture.h

```diff
@@ -87,6 +87,26 @@ typedef int32_t     sword_t;
  */
 #define WORD_ALIGNED __attribute__((aligned(ARCHITECTURE_WORD_BYTES)))
 
+/**
+ * @brief   Smallest number an uword_t can hold
+ */
+#define UWORD_MIN                   0
+
+/**
+ * @brief   Highest number an uword_t can hold
+ */
+#define UWORD_MAX                   ((1ULL << ARCHITECTURE_WORD_BITS) - 1)
+
+/**
+ * @brief   Smallest number an sword_t can hold
```

Comment:
```suggestion
 * @brief   Smallest number a sword_t can hold
```

Reply:
"an" is actually right here, because you pronounce `sword_t` as "es-word-tea`